CONTRIBUTING.md

```diff
@@ -43,12 +43,12 @@ Get started
 5. Check your environment
    eg. $`python --version` -> see that it uses Python 3.13.*
    eg. $`which python` -> see that it matches you python executable in the venv you have created: $```echo `pwd`/.venv/bin/python```
-6. Install the package: $`pdm install`
-7. Add `export PYTHONPATH="$PYTHONPATH:./src"`
-8. Check your environment: $`./manage.py env --check` and configure the missing variables.
+5. Install the package: $`uv sync`
+6. Add `export PYTHONPATH="$PYTHONPATH:./src"`
```

Comment:
PYTHONPATH probably needs to contain also /tests after recent change of tests structure right?